syntax = "proto3";
package dpi;

message MemoryResourceResponse {
    uint64 offset = 1;
    uint32 return = 2;
}
